makefile

```diff
@@ -0,0 +1,15 @@
+QKCDIR = `pwd`/qkchash
+GRPCDIR = `pwd`/quarkchain/cluster
```

Comment:
1. prefer to leaving all the generated files to `quarkchain/generated`
2. seems we don't need `pwd`?
3. also since the directory path and make command is short, I think we don't need to have variables for them

Reply:
also https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html

Reply:
Done.